### PR TITLE
Update master with PRDOPSSOC-055

### PR DESCRIPTION
**Pull request by Witchblade101**
This PR updates pysiaf with the new PRD release PRDOPSSOC-055. SIAF data has been added to the `prd_data` directory and removed from the `pre_delivery_data` directory.

### CODE REVIEW

File: generate/generate_nircam.py [hunk 1]
```diff
@@ -388,7 +388,7 @@
 
         create_jira_plots = True
         if create_jira_plots:
-            # # make figures for JWSTSIAF-184 Jira ticket
+            # # make figures for JWSTSIAF-245 Jira ticket
```
Comment (mfixstsci) [inline generate/generate_nircam.py:391]: Were these changes already merged into `siaf-updates` already?

**Review by mfixstsci — COMMENTED** [review]
I see the changes to the `xml` and the aperture text files. I see you deleted the `pre-delivery` data in this PR.

File: generate/generate_nircam.py (hunk 1 above)
Comment (Witchblade101) [reply generate/generate_nircam.py:391]: Those changes were from https://github.com/spacetelescope/pysiaf/pull/268/files

**Review by mfixstsci — APPROVED** [review]
File changes are as expected. Asked @Witchblade101 about code changes and they all check out.

**Pull request merged**